.github/workflows/check-repos.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           GITHUB_TOKEN: ${{ steps.github_app_token.outputs.token }}      
         run: |
           npm install -g meta
-          ./scripts/update.sh
+          ./bin/check-repos.sh
```

Comment:
It would be fun to unit test this script.

Reply:
I plan to rewrite this nonsense ;)